src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -32,6 +32,12 @@ public static bool IsPlaceholderModeEnabled(this IEnvironment environment)
             return environment.GetEnvironmentVariable(AzureWebsitePlaceholderMode) == "1";
         }
 
+        public static bool IsEasyAuthEnabled(this IEnvironment environment)
+        {
+            bool.TryParse(environment.GetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled), out bool isEasyAuthEnabled);
+            return isEasyAuthEnabled;
```

Comment:
non-blocking since this works, but a somewhat more correct way to write this is:

```csharp
return
    bool.TryParse(environment.GetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled), out bool isEasyAuthEnabled) &&
    isEasyAuthEnabled;
```

This is more correct because you're not depending on the output value behavior of `TryParse` when parsing fails.